### ISSUE TITLE
in mac always asking for java

### ISSUE THREAD

**Issue by Hanzala717** [appdictive/ADBRemoteConnect#17]
not working in mac always asking for java either I installed java 8 rev 201 but not working 

"**To use the “java” command-line tool you need to install a JDK.**"

**Comment by berkayturanci** [comment]
Did you try this: https://stackoverflow.com/a/46828380/1847645

**Comment by Hanzala717** [comment]
yes Java issue is resolved but plugin is not working
click on it respond nothing

**Comment by berkayturanci** [comment]
Follow the link; https://github.com/appdictive/ADBWiFiConnect/issues/5#issuecomment-474788818

**Comment by Hanzala717** [comment]
that is for windows

**Comment by berkayturanci** [comment]
> that is for windows

No, I used it in macOS. It contains .jar file which both MacOs and Windows supports.

**Comment by RRGT19** [comment]
I'm experiencing the same issue. I tried both solutions mentioned before, [this](https://stackoverflow.com/questions/44009058/even-though-jre-8-is-installed-on-my-mac-no-java-runtime-present-requesting-t/46828380#46828380) and [this](https://github.com/appdictive/ADBWiFiConnect/issues/5#issuecomment-474788818) and none of them worked. Clicking on the green icon doesn't open anything.
